### PR TITLE
tests: zbus: modify test identifiers to match component

### PR DESCRIPTION
**Pull request by nashif**
Use message_bus as component in all zbus tests for better organisation
and to be able to select them in test plans.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

File: tests/subsys/zbus/dyn_channel/testcase.yaml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 tests:
-  dyn_channel.static_and_dynamic_channels:
-    build_only: false
+  message_bus.dyn_channel.static_and_dynamic_channels:
```
Comment (stephanosio) [inline tests/subsys/zbus/dyn_channel/testcase.yaml:2]: If the goal is to categorise Zbus tests, why not use `zbus` instead of `message_bus`?

Comment (rodrigopex) [reply tests/subsys/zbus/dyn_channel/testcase.yaml:2]: Maybe zbus make more sense for this case.

**Review by rodrigopex — COMMENTED** [review]
I agree with @stephanosio. Maybe zbus would be better for consistency and stating the name.

File: tests/subsys/zbus/dyn_channel/testcase.yaml (hunk 1 above)
Comment (nashif) [reply tests/subsys/zbus/dyn_channel/testcase.yaml:2]: The idea is to have the feature, not the name of the subsystem in zephyr, thats why we have `testing.ztest`
 and not `ztest`, etc., could be changed to `message_bus.zbus`, I am fine with that.

Comment (rodrigopex) [reply tests/subsys/zbus/dyn_channel/testcase.yaml:2]: > message_bus.zbus

Understood. It makes sense. `message_bus.zbus` is good.

**Review by rodrigopex — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Pull request merged**